sshcode.go

```diff
@@ -608,3 +625,73 @@ func parseGCPSSHCmd(instance string) (ip, sshFlags string, err error) {
 
 	return strings.TrimSpace(userIP), sshFlags, nil
 }
+
+// gitbashMountPoints returns all mount points in a msys2 system and then trunicates
+// the mount point for `/` from the path provided (dir), this is done to fix an
+// issue with how msys2 provides file paths to go, example. When you feed a filepath
+// as user input to go, `command /opt` for example, msys2 will pass `C:\<path to msys2 directory>\opt`
+// This causes an issue whith this program where when you run `sshcode.exe user@server /opt`,
+// GO and msys2 will attempt to pass `C:\<path to msys2 directory>\opt` to the server.
+// This function is used to prevent this.
+func gitbashWindowsDir(dir string) string {
```

Comment:
Your `default.env` file you pastebinned contains the following env vars:
```
MSYSTEM_PREFIX=/mingw64
MSYSTEM=MINGW64
MINGW_PREFIX=/mingw64
MINGW_CHOST=x86_64-w64-mingw32
```

Could we use these instead? This function is very complicated

Reply:
Unfortunately no, as there is no environment variable that directly points to the install Dir.
Those envs are used for the package manager and $PATH variables.